test/aave-v3/TestSupplyVault.t.sol

```diff
@@ -667,4 +667,39 @@ contract TestSupplyVault is TestSetupVaults {
         assertApproxEqAbs(uint256(userReward1_1), expectedTotalRewardsAmount, 10000);
         assertApproxEqAbs(uint256(userReward1_1), userReward1_2, 1);
     }
+
+    function testTransferAndClaimRewards() public {
```

Comment:
Can you add the other transfer tests please?

Reply:
WDYM? Not sure what tests I'm missing here since i dont see any other comparable tests on compound.

Reply:
Hmm ok I was refering to `testTransfer` and `testTransferFrom` but this on the `main` branch. I should add it to `upgrade-0`

Reply:
I'd like to see the changes to use RAYs merged before we add those tests, as some of the tests you are referencing will need to be revised when that is merged.

Reply:
Makes sense. 

I'll still pull `main` from `upgrade-0` though

Reply:
Can you reproduced the tests [here](https://github.com/morpho-dao/morpho-tokenized-vaults/blob/upgrade-0/test/aave-v2/TestSupplyVaultBase.t.sol) please @pakim249CAL ?